vlib/v/checker/tests/ambiguous_function_call.vv:2:2: error: ambiguous call to: `foo1`, may refer to fn `foo1` or variable `foo1`
    1 | fn foo1(foo1 int) {
    2 |     foo1(foo1 + 1)
      |     ~~~~~~~~~~~~~~
    3 | }
    4 |
vlib/v/checker/tests/ambiguous_function_call.vv:7:2: error: ambiguous call to: `foo2`, may refer to fn `foo2` or variable `foo2`
    5 | fn foo2() {
    6 |     foo2 := 1
    7 |     foo2(foo2)
      |     ~~~~~~~~~~
    8 | }
    9 |
vlib/v/checker/tests/ambiguous_function_call.vv:7:7: error: expected 0 arguments, but got 1
    5 | fn foo2() {
    6 |     foo2 := 1
    7 |     foo2(foo2)
      |          ~~~~
    8 | }
    9 |
Details: have (int)
         want ()
